packages/strapi-design-system/src/Layout/HeaderLayout.js

```diff
@@ -105,7 +101,11 @@ export const BaseHeaderLayout = React.forwardRef(
       );
     }
 
-    const FlexElement = props?.ellipsis ? TitleFlex : Flex;
+    let flexProps = {};
+
+    if (props?.ellipsis) {
+      flexProps.minWidth = 0;
+    }
```

Comment:
Is there a reason you opted to do it this way? As opposed to `minWidth={ ellipsis ? 0 : undefined }`?

Also do we need to conditionally add it? Does it actually cause any issues if we leave as is? 🤔 Feels very flakey to have to do it this way.

Reply:
I have added this option just to prevent any weird behavior in the other places where we use HeaderLayout but yes, I can add the prop every time and in the QA phase check every HeaderLayout in Strapi

Reply:
I did it, no regression errors, maybe, in a future a suggestion is to add the ellipsis prop even in other places where we use the HeaderLayout because can prevent some weird behaviours with long titles or subtitles

Reply:
I think in the future it would be nicer to use composition so you have more control over what you put in where e.g. 

```jsx
<HeaderLayout.Root>
	<HeaderLayout.Action><MyIconButton /></HeaderLayout.Action>
	<HeaderLayout.Title>Hello</HeaderLayout.Title>
	<HeaderLayout.Subtitle>there</HeaderLayout.Subtitle>
</HeaderLayout.Root>
```